deployment.yaml

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project.
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: egeria-connector
+  name: egeria-connector-service
+spec:
+  type: NodePort
+  ports:
+    - port: 9443
+      name: egeria
+      targetPort: 9443
+      nodePort: 30000
+      protocol: TCP
+    - name: debug
+      port: 5005
+      targetPort: 5005
+      nodePort: 30001
+  selector:
+    app.kubernetes.io/name: egeria-connector
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  annotations:
+    sas.com/certificate-file-format: jks
+  creationTimestamp: null
+  name: egeria-connector-ss
+  labels:
+    app.kubernetes.io/name: egeria-connectors
+spec:
+  replicas: 1
+  serviceName: egeria-connector-service
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: egeria-connector
+  template:
+    metadata:
+      annotations:
+        sas.com/certificate-file-format: jks
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/name: egeria-connector
+    spec:
+      imagePullSecrets: []
+      containers:
+        - name: egeria
+          image: odpi/egeria-connector-sas-viya
+          env:
+            - name: JAVA_DEBUG
+              value:  "true"
```

Comment:
Did you want this as a default? It will leave the debug port accessible (also see ref to port 5005)

Reply:
Good catch, I can remove this in the next push.